iotqatools/simulators/listen/listen.js

```diff
@@ -21,36 +21,77 @@ For those usages not covered by the GNU Affero General Public License
 please contact with::[iot_support@tid.es]
  */
 
-//script version
-var version="0.0.3"
+
+// requirements
+var dateFormat = require('dateformat');
+var moment = require('moment');
+
 
 //verbosity flag
 var vm=false;
+
 //silence flag
 var silence=false;
 
+//timestamp flag
+// expected input to calculate the diff in format 2015-10-28T13:51:32+0100
+/* {
+    "subscriptionId": "56309dbebc0ae94ca4f7f8ea",
+    "originator": "localhost",
+    "contextResponses": [
+        {
+            "contextElement": {
+                "type": "xxx",
+                "isPattern": "false",
+                "id": "aaa",
+                "attributes": [
+                    {
+                        "name": "TimeInstant",
+                        "type": "ISO8601",
+                        "value": "2015-10-28T13:51:32+0100"
```

Comment:
Is the date parsing logic strong enough to cope with miliseconds in the case the value includes it? It will work if timezone is not included?